scripts/quota_log.py

```diff
@@ -35,12 +47,19 @@ def disk_quota():
 
 def cpu_hours():
     current_time = datetime.datetime.now()
-
-    # script that runs on uppmax
-    uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    try:
+        # script that runs on uppmax
+        uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
+    except Exception, e:
+        logging.error(e.message)
+        exit()
```

Comment:
except a well formatted logging gives you date, place and running script.

On Mon, Sep 14, 2015 at 4:22 PM, Phil Ewels <notifications@github.com>
wrote:

> In scripts/quota_log.py
> <https://github.com/SciLifeLab/scilifelab/pull/482#discussion_r39399028>:
>
> > @@ -35,12 +47,19 @@ def disk_quota():
> >
> >  def cpu_hours():
> >      current_time = datetime.datetime.now()
> > -
> > -    # script that runs on uppmax
> > -    uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
> > +    try:
> > +        # script that runs on uppmax
> > +        uq = subprocess.Popen(["/sw/uppmax/bin/projinfo", '-q'], stdout=subprocess.PIPE)
> > +    except Exception, e:
> > +        logging.error(e.message)
> > +        exit()
>
> Or even just remove the try / except block? As the logging is going to the
> terminal anyway, so no difference to the python raise really.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/scilifelab/pull/482/files#r39399028>.
>